beheer.md

```diff
@@ -22,6 +22,40 @@ In dit document worden allerlei procedures en overzichten beschreven die van bel
 
+## Beheerproces
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
+Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+1. beantwoorden van vragen;
+2. het oplossen van problemen.
+
+Doorontwikkeling van de standaard hoort daar dus niet bij.
+
+*Beantwoorden van vragen*
+Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn twee kanalen beschikbaar. Ten eerste het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace 'VNG API Community' en ten tweede het eveneens specifiek voor de standaard beschikbaar gestelde discussie forum.
+
+_Beschouwen we de discussie voorzieningen op GitHub ook als een goed alternatief voor de discussie fora?
+Zo ja dan moeten we n.m.m. een keuze tussen beide maken. Het monitoren van meerdere omgevingen op vragen/wijzigingsverzoeken moet zo veel mogelijk voorkomen worden._
+
+Slack leent zich niet zo goed voor het gestructureerd voeren van een discussie aangezien in het kanaal meerdere onderwerpen door elkaar heen kunnen gaan lopen en (in de basis variant) historie slechts tot een x aantal reacties beperkt blijft. Slack is vooral geschikt voor het snel stellen en beantwoorden van enkelvoudige vragen.
+
+Voor het diepgaand bediscusieren van issues en ter sprake stellen van problemen in de standaard waarbij de discussies een lange(re) tijd beschikbaar moeten blijven kan beter het discussieforum worden gebruikt.
+
+Zodra het van belang is om een in Slack gevoerde discussie toch voor langere tijd te bewaren kunnen de beheerders de betreffende discussie alsnog onderbrengen op het discussie platform. Indien de discussie vragen over het gebruik en toepassing van de standaard betreft dienen de beheerders zich steeds af te vragen of het niet beter is de documentatie van de standaard zo aan te passen dat eenzelfde vraag in de toekomst voorkomen wordt.
+
+Vragen in beide kanalen moeten binnen xx dagen beantwoord zijn.
+
+*Probleemoplossing*
+De initiator voor probleemoplossing van een standaard is altijd een in GitHub ingebracht issue. 
+Indien het wijzigingsverzoek via Slack binnenkomt maakt de beheerder een issue aan in GitHub. 
+
+Na opvoering van het issue beoordeelt de beheerder of het een valide issue is. Indien dat niet het geval is wordt dat gemeld in het issue met de reden daarvoor waarna de melder enkele weken (_hoeveel?_) de tijd krijgt om de reden te weerleggen en zijn verzoek nog nader toe te lichten.
```

Comment:
Terecht punt. Issue aanmaken om dit van een waarde te voorzien?

Reply:
Ik zou zeggen het oorspronkelijke issue sluiten en de melder vragen om het issue nogmaals maar dan met nadere toelichting in te dienen. Dan blijven we ook niet zitten met losse eindjes en issues waarvan we niet zeker weten of ze nog in behandeling genomen moeten worden

Reply:
@michielverhoef dat zou betekenen dat we voor het bekijken van de historie van een verzoek meerdere issues moeten bekijken. Ik voel er meer voor om het in 1 issue te houden en dat te sluiten als de termijn verstreken is om ons besluit te weerleggen. Zie issue #95.